packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -40,6 +43,34 @@ fn api_documentation() -> Vec<openapi::OpenApi> {
     ]
 }
 
+async fn read_from_store<'pool, 'id: 'pool, T, O, P, I>(
+    pool: &'pool P,
+    identifier: I,
+) -> Result<O, StatusCode>
+where
+    P: StorePool,
+    P::Store<'pool>: crud::Read<'id, I, T, Output = O>,
+    I: Send + 'id,
```

Comment:
It's probably possible to remove the `O` parameter, but I wasn't able to find the right bounds for `Read::Output`. In any case, this is more readable.